src/main.rs

```diff
@@ -24,15 +47,20 @@ fn init_heap() {
         ALLOCATOR.init(HEAP.as_mut_ptr() as *mut u8, HEAP_SIZE);
     }
 }
-{% endif %}
+{% endif -%}
 #[entry]
 fn main() -> ! {
     {%- if alloc %}
     init_heap();
-    {%- endif %}
+    {%- endif   %}
     let peripherals = Peripherals::take();
+    {% if arch == "xtensa" and wifi-%}
```

Comment:
Whats the purpose of this if, can we remove it?

Reply:
To make Clippy happy, the difference is only the `mut` but Clippy will complain if its present and not used.